_posts/ianmoore/2016-01-14-ianmoore-turtles.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+author: touchwick
+title: Ian's Turtle Exercise
+---
+
+This is the first program I'm embedding:
+
+  <iframe src="https://trinket.io/embed/python/19de38af1f" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+And this is the second:
+
+  <iframe src="https://trinket.io/embed/python/c78e584bff" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+The above programs were relatively straightforward to build. The second, obviously, pulls heavily from the first. It just has a sad story on top of it. I faced some issues creating punctuation with the ```print``` command, particularly on line 83: ```print "I made this flower for you,", tina_yourname,"."```. This command creates a space between the value entered for tina_yourname and the period, which is obviously not grammatically correct. Whoops!
```

Comment:
Haha nice!  Very clever.

Great question about punctuation. We'll get more into this in a but, but when you use commas with print it does autoatically insert spaces, treating each thing as its own little blurb to be printed in a row.  To get it to not insert spaces you can instead concatenate (join) the text with a `+`.  In that case `print` only sees one unified blob of text.